PowerShellTools/DebugEngine/ScirptProgramNode.cs

```diff
@@ -172,7 +172,7 @@ public int CanDetach()
         public int Detach()

```

Comment:
Yes, this is called when a user hits the stop button while attached to a process.